src/polycubed/src/CMakeLists.txt

```diff
@@ -76,6 +78,7 @@ set(polycubed_libraries
   viface
   dl
   pistache
+  prometheus-cpp-core
```

Comment:
So, prometheus-cpp-core will be compiled as a library and linked statically, isn't it? I would write it in the main readme (the one that summarizes the external libs used).
Instead, it seems that jsoncons is compiled within Polycube, isn't it? I would write it in the main readme as well.

Reply:
Done :+1: